src/Controller/QuestionTemplateController.php

```diff
@@ -76,19 +76,19 @@ public function deleteQuestion(Request $request, array $requestAttributes): Resp
         $this->questionTemplateService->delete($id);
         $body = Stream::createFromString('');
         $response = new Response($body, '1.1', 301, '');
-        $response = $response->withHeader('Location', 'http://local.quiz.com/listQuestions');
+        $response = $response->withHeader('Location', 'http://local.quiz.com/listQuestions?page=1');
 
         return $response;
     }
 
     public function getQuestions(Request $request, array $requestAttributes): Response
     {
-        $questionRepo = $this->repositoryManager->getRepository(QuestionTemplate::class);
-        // get filters and pagination from request
-        $criteria = new Criteria();
-        $questions = $questionRepo->findBy($criteria);
-        $questions = ['questions' => $questions];
-        return $this->renderer->renderView('admin-questions-listing.phtml', $questions);
+        $page = (int)$request->getParameter('page');
```

Comment:
ai putea sa faci metoda `getParameter()` sa fie mai inteligenta un pic. Ia arunca un ochi pe aici https://github.com/symfony/http-foundation/blob/master/ParameterBag.php#L74-L77 . Daca faci ceva de genu, nu mai ai nevoie sa adaugi pagina 1 in toate redirecturile tale.